crates/eips/src/eip7702/auth_list.rs

```diff
@@ -0,0 +1,119 @@
+use core::ops::Deref;
+
+#[cfg(not(feature = "std"))]
+use alloc::vec::Vec;
+use alloy_primitives::{keccak256, Address, ChainId, Signature, SignatureError, B256};
+use alloy_rlp::{BufMut, Decodable, Encodable, Header, RlpEncodable};
+
+use super::constants::MAGIC;
+
+/// An EIP-7702 authorization list.
+pub type AuthorizationList = Vec<Authorization>;
```

Comment:
I'd like to remove this aliases, imo aliases for vec types are always bad

Reply:
ok